### PR TITLE
Move to Catalyst`s docker image

### PR DESCRIPTION
**Pull request by vaklyuenkov**
Looks like versions of Pandas and tqdm have conflict like that: https://github.com/tqdm/tqdm/issues/555.

To avoid it added version of tqdm to requirement, but better fix it in image itself.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,31 +1,9 @@
-FROM pytorch/pytorch:1.0-cuda10.0-cudnn7-devel
+FROM catalystteam/catalyst:19.10
```
Comment (TezRomacH) [inline docker/Dockerfile]: let's make it `catalystteam/catalyst:19.10-fp16` be default

File: requirements/requirements_local.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ catalyst==19.10
 albumentations==0.4.0
 opencv-python==4.0.0.21
 jinja2
+tqdm>=4.33.0
```
Comment (TezRomacH) [inline requirements/requirements_local.txt]: to catalyst ?

Comment (vaklyuenkov) [reply requirements/requirements_local.txt]: if I undestand your question correctly using tqdm>=4.33.0 in main repo Catalist is the best conflict resolution

Comment (TezRomacH) [reply requirements/requirements_local.txt]: Yeah, let's fix it on the framework level

File: docker/Dockerfile (hunk 1 above)
Comment (vaklyuenkov) [reply docker/Dockerfile]: tested and added

File: requirements/requirements_local.txt (hunk 2 above)
Comment (vaklyuenkov) [reply requirements/requirements_local.txt]: After that I'll remove tqdm>=4.33.0 from req's

Comment (vaklyuenkov) [reply requirements/requirements_local.txt]: https://github.com/catalyst-team/catalyst/pull/480

Comment (vaklyuenkov) [reply requirements/requirements_local.txt]: but one need also update docker images

**Pull request merged**